vsts_ci/win32/continuous-build-win32.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+  - script: |
```

Comment:
Why do you separate the tox for Python 2 and 3? For Linux (https://github.com/Azure/iotedgehubdev/pull/88/files#diff-3a88ca3f111f0142018125fb3ea8226f), you only run it under Python 3. tox will handle both Python 2 and 3.

Reply:
no. There is no 2 and 3 in win1803 and no env variable for python

Reply:
Use `choco` to install both Python 2 and Python 3, and tox will be able to find them automatically as they are installed in the standard prefix.

Reply:
updated